src/main/resources/view/DarkTheme.css

```diff
@@ -351,10 +351,49 @@
 }
 
 #tags .label {
-    -fx-text-fill: white;
-    -fx-background-color: #3e7b91;
-    -fx-padding: 1 3 1 3;
+    -fx-padding: 2 5 2 5;
     -fx-border-radius: 2;
-    -fx-background-radius: 2;
+    -fx-background-radius: 8;
     -fx-font-size: 11;
 }
+
+#tags .red {
+    -fx-text-fill: white;
+    -fx-background-color: #F70D1A;
+}
+
+#tags .yellow {
+    -fx-text-fill: black;
+    -fx-background-color: #F4F00D;
+}
+#tags .grey {
+    -fx-text-fill: black;
```

Comment:
Grey background and black text seems to have low contrast, maybe change text to white instead? [Here](https://webaim.org/resources/contrastchecker/) is a website that helps calculate contrast ratio, might be useful.

Reply:
Cool, thanks! I'll give it a try